aopy/analysis/connectivity.py

```diff
@@ -0,0 +1,238 @@
+import numpy as np
+from tqdm.auto import tqdm
+import multiprocessing as mp
+
+from . import base
+from .. import data as aodata
+from .. import precondition
+from . import accllr
+
+def get_acq_ch_near_stimulation_site(stim_site, stim_layout='Opto32', electrode_layout='ECoG244', 
+                                 dist_thr=1, return_idx=False):
+    '''
+    Get acquisition channels near a stimulation site. Warning: this function returns channel
+    numbers, which sometimes are 1-indexed. Set the return_idx flag to True to get the channel
+    indices as well.
```

Comment:
Is there a sop or other function that addresses ecog channel mapping in more detail? Could you potentially link/mention that here so I know where to go to understand the channel mapping?

Reply:
added a link! we try not to rely on SOPs since it's a public repo